README.md

```diff
@@ -19,6 +19,10 @@ Run ```node ./scripts/kick-off.js``` from project's rootpath to remove the exist
 Before running the app, make sure you must have a postgres db created. Then, set the `$NODE_API_DB_URL` environmental variable. It should look something like: `postgres://username:password@host:port/databasename`.
 For more information feel free to glance at the [`app/orm.js`](https://github.com/Wolox/express-js-bootstrap/blob/master/app/orm.js#L6) file.
 
+### Migrations
+
+To create a migration, run ```sequelize migration:create --name="my-migration-name"```. To run migrations, run ```npm run migrations```.
```

Comment:
@gonziesc I think he's talking about the `migration-create` script you added In the README. Also, what about that `name` param? Migrations must have some timestamp in its name so that they are run in the same order they were created